src/ert/scheduler/lsf_driver.py

```diff
@@ -440,6 +452,14 @@ async def poll(self) -> None:
                     f"bjobs gave returncode {process.returncode} and error {stderr.decode()}"
                 )
             bjobs_states = _parse_jobs_dict(parse_bjobs(stdout.decode(errors="ignore")))
+            bjobs_exec_hosts = parse_bjobs_exec_hosts(stdout.decode(errors="ignore"))
+
+            for jobid, exec_hosts in bjobs_exec_hosts.items():
+                if self._jobs[jobid].exec_hosts == "-":
+                    logger.info(
+                        f"Realization {self._jobs[jobid].iens} was executed on host: {exec_hosts}"
```

Comment:
I guess "was executed" is not strictly correct, as of the time this message is sent, it is not done yet. "is being executed"?

Reply:
What about "was assigned to" ? Otherwise i'm fine with the other alternative.